modules/topology/models/Domain.php

```diff
@@ -63,6 +64,7 @@ public function attributeLabels()
             'id' => 'ID',
             'name' => Yii::t('topology', 'Name'),
             'color' => Yii::t('topology', 'Color'),
+            'grouped_nodes' => Yii::t('topology', 'Grouped Nodes'),
```

Comment:
The translation for this has been added at:
modules/topology/messages/pt-BR/topology.php
?

Reply:
Now it is.